tests/Feature/Repositories/TaskRepositoryTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Tests\Feature\Repositories;
+
+use App\Models\Task;
+use App\Models\User;
+use App\Repositories\CommentRepository;
+use App\Repositories\TaskRepository;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Collection;
+use Tests\TestCase;
+
+class TaskRepositoryTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_create_task_is_successful(): void
+    {
+        $user = new User();
+        $user->setAttribute('name','test');
+        $user->setAttribute('email', 'test@gmail.com');
+        $user->setAttribute('password', '2233');
+        $user->save();
+
+        $repository = new TaskRepository;
+
+        $result = $repository->createTask([
+            'name' => 'task',
+            'description' => 'task one',
+            'status' => 'ongoing'
+        ]);
+
+        $repository2 = new CommentRepository;
+
+        $result2 = $repository2->create([
+            'title' => 'new title',
+            'body' => 'new body'
+        ], $result);
+
+        $this->assertDatabaseHas('tasks', [
+            'name' => $result->getAttribute('name'),
+            'description' => $result->getAttribute('description'),
+            'status' => $result->getAttribute('status')
+        ]);
+
+        $this->assertDatabaseHas('comments', [
+            'title' => 'new title',
+            'body' => 'new body'
+        ]);
+    }
+
+    public function test_index_task_is_successful(): void
+    {
+        $user = new User();
+        $user->setAttribute('name','test');
+        $user->setAttribute('email', 'test@gmail.com');
+        $user->setAttribute('password', '2233');
+        $user->save();
+
+        $repository = new TaskRepository;
+
+        $task = new Task();
+        $task->setAttribute('name', 'task1');
+        $task->setAttribute('description', 'task-1');
+        $task->setAttribute('status', 'completed');
+        $task->save();
```

Comment:
Write one more task. Index should get a collection of tasks. So our test should cover more than one task as well.